packages/eas-cli/src/build/build.ts

```diff
@@ -229,9 +235,15 @@ export function handleBuildRequestError(error: any, platform: Platform): never {
   throw error;
 }
 
-async function uploadProjectAsync<TPlatform extends Platform>(
-  ctx: BuildContext<TPlatform>
-): Promise<string> {
+async function uploadProjectAsync<TPlatform extends Platform>({
+  ctx,
+}: {
+  ctx: BuildContext<TPlatform>;
+  generateMetadataFile: boolean;
```

Comment:
We should either use `generateMetadataFile` or remove it from function arguments